.github/dockerfiles/ov_build/ubuntu_22_04_x64/Dockerfile

```diff
@@ -24,9 +24,18 @@ RUN apt-get update && \
         tzdata \
         libtbb2 \
         # Pythons \
+        python3.9-dev \
+        python3.9-venv \
+        python3.9-distutils \
+        python3.10-dev \
+        python3.10-venv \
+        python3.10-distutils \
         python3.11-dev \
         python3.11-venv \
         python3.11-distutils \
+        python3.12-dev \
+        python3.12-venv \
+        python3.12-distutils \
```

Comment:
I think we need to use manylinux docker image to build python versions. That docker image contains all the required python binaries for all OpenVINO supported versions.

Reply:
The manylinux introduction is tracked by a separate ticket (148719), @mryzhov might bring some insight on the progress/POC he did some time ago. 

With this PR I was trying to match the test scope in Jenkins. I suggest we proceed with this approach as it introduces the tests in a separate workflow + the action for finding the right Python wheels, they could be adapted later when we have the manylinux introduced.